pypi_formula_mappings.json

```diff
@@ -267,6 +267,9 @@
   "ocrmypdf": {
     "exclude_packages": ["pillow"]
   },
+  "onlykey-agent": {
+    "exclude_packages": ["six"]
```

Comment:
I think you also need `cython` here.

Reply:
added in f5ad421c24b031e42849b3968696ec7c944a269d